docs/upgrade.md

```diff
@@ -168,6 +169,13 @@ In Marionette 3, the HTML will be:
 </div>
 ```
 
+### Arguments for Lifecycle Events
+
+The arguments to all lifecycle events were completely changed. All events will
```

Comment:
Cool, I'll just list the ones that did change and direct the reader to the documentation for everything so people can check it out for themselves.